classes/output/core_renderer.php

```diff
@@ -463,4 +463,63 @@ public function block(\block_contents $bc, $region) {
 
         return $this->render_from_template('core/block', $context);
     }
+
+    /**
+     * Allow plugins to provide some content to be rendered in the navbar.
+     * The plugin must define a PLUGIN_render_navbar_output function that returns
+     * the HTML they wish to add to the navbar.
+     *
+     * @return string HTML for the navbar
+     */
+    public function navbar_plugin_output() {
```

Comment:
-> this should be moved into lib.php using the render_navbar_output hook.

Reply:
Thanks for the feedback, fixed!